pkg/e2e/fixtures.go

```diff
@@ -545,3 +593,31 @@ func EnsureServiceAttachment(s *Sandbox, saName, svcName, subnetName string) (*s
 func DeleteServiceAttachment(s *Sandbox, saName string) error {
 	return s.f.SAClient.Delete(s.Namespace, saName)
 }
+
+// GetL4LBForSvcParams extract GCP resources name from NetLB service annotations
+func GetL4LBForSvcParams(s *Sandbox, svcName string) (*fuzz.L4LBForSvcParams, error) {
+	svc, err := s.f.Clientset.CoreV1().Services(s.Namespace).Get(context.TODO(), svcName, metav1.GetOptions{})
+	params := fuzz.L4LBForSvcParams{
+		VIP:                   svc.Status.LoadBalancer.Ingress[0].IP,
+		SvcName:               svcName,
+		BsName:                svc.Annotations[annotations.BackendServiceKey],
+		FwrName:               svc.Annotations[annotations.TCPForwardingRuleKey],
+		HcName:                svc.Annotations[annotations.HealthcheckKey],
+		HcFwRuleName:          svc.Annotations[annotations.FirewallRuleForHealthcheckKey],
+		FwRuleName:            svc.Annotations[annotations.FirewallRuleKey],
+		ExternalTrafficPolicy: string(svc.Spec.ExternalTrafficPolicy),
+	}
+	return &params, err
+}
+
+func CompareHealthChecks(s *Sandbox, svc1Name, svc2Name string) error {
```

Comment:
this should be in helpers rather than fixtures

Reply:
done